source/includes/fact-write-blocking-enable.rst

```diff
@@ -0,0 +1,7 @@
+When write-blocking is enabled, ``mongosync`` blocks writes:
+
+- On the destination cluster during replication
+- On the source cluster while committing
+
+To enable write-blocking, use the :ref:`start API <c2c-api-start>`
+to set ``enableUserWriteBlocking`` to ``true``.
```

Comment:
Reading this, my understanding is that you can't modify write-blocking after a sync operation has started. Is this true? If so, it might be worth explicitly mentioning this behavior.